### PR TITLE
Robo limbs proper malfunction and slight damage reduction buff.

### PR DESCRIPTION
**Pull request by Swagile**
## About The Pull Request

Makes it so you have a proper malfunction threshold with robo limbs and slight buff from 0.66 to 0.50 reduction to damage.

## Why It's Good For The Game

Currently, robo limbs penalize you too much. They malfunction randomly (with no way to fix this except full repair), do not benefit from autodoc / chems, and on top of this require you to restructure your entire setup around it (welder, jaeger helmet or welding helmet if you use PAS, and cables).

This PR seeks to alleviate the worst aspect of the robo limbs (random malfunctions) by removing them entirely, and buffing the amount of damage robo limbs take. This still forces a permanent condition on marines (since they still have to re-order their gear around it), but it also makes it not a "im delimbed, might as well cryo" moment.

Reason why it is important to keep this aspect of the game still is that delimbs are one of the only ways to stop competent marines from constantly pushing; perma kills are basically gone, broken bones are fixed by valk, B18, or splinting.

This is more of a "I am not really good enough at coding to fix this issue entirely but I am good enough to fix it with a bandaid" kinda situations.

## Changelog
:cl:
balance: Robo limbs now malfunction at a proper threshold, and do it at a lower probability than fleshy limbs.
balance: Robo limbs take more punishment; from 0.66 reduction to 0.50 reduction to brute and burn damage.
/:cl:

### CODE REVIEW

File: code/modules/organs/limbs.dm [hunk 1]
```diff
@@ -1038,7 +1038,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 			var/emote_scream = pick("screams in pain and", "lets out a sharp cry and", "cries out and")
 			owner.emote("me", 1, "[(owner.species && owner.species.species_flags & NO_PAIN) ? "" : emote_scream ] drops what they were holding in their [hand_name]!")
 	if(is_malfunctioning())
-		if(prob(10))
+		if(prob(0))
```
Comment (SplinterGP) [inline code/modules/organs/limbs.dm:1041]:
```suggestion
		if(prob(5))
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]
do the thing i told you at discord

**Comment by Swagile** [comment]
done and thanks @SplinterGP for teaching me how to add a proper threshold to malfunction; the no prob(0) was a bandaid fix cause i didn't know how to fix it

**Review by TiviPlus — APPROVED** [review]

**Review by SplinterGP — APPROVED** [review]

**Comment by DerrickChamberlain** [comment]
Is applying cable only suppose to work when you do it, or is it a bug that other people can not apply the cables to robo limbs?

**Comment by Swagile** [comment]
> 
> 
> Is applying cable only suppose to work when you do it, or is it a bug that other people can not apply the cables to robo limbs?

this PR doesn't affect that (or shouldn't, since it handles malfunctions and reduction to damage)

as to the answer to that, no idea

**Comment by DerrickChamberlain** [comment]
> > Is applying cable only suppose to work when you do it, or is it a bug that other people can not apply the cables to robo limbs?
> 
> this PR doesn't affect that (or shouldn't, since it handles malfunctions and reduction to damage)
> 
> as to the answer to that, no idea

Oh I know. I was just asking. It sucks you can not use cable of other peoples robo limbs they have to apply them. If me apply cables does work, there needs to be some feed back.

**Pull request merged**